contracts/CommonHealthCheck.sol

```diff
@@ -94,37 +73,42 @@ contract CommonHealthCheck {
     }
 
     function check(
+        address strategy,
```

Comment:
we should overload this to have two methods called `check` one thats used by vault now, and the old one used in existing strategies, we want to make this the global healthcheck for both vaults and strategies and this new version will become `health.ychad.eth` so we have a single contract to manage this. Both the existing healthchecks and this new vault checks should be supported.

Something to test in the test suite that it can support strategies healthchecks for old versions.

Reply:
Tested and tagged for reference here:
https://github.com/yearn/yearn-vaults/compare/v0.4.3...v0.4.3-1